src/lib/gssapi/krb5/inq_cred_oid.c

```diff
@@ -0,0 +1,62 @@
+/* -*- mode: c; c-basic-offset: 4; indent-tabs-mode: nil -*- */
+/*
+ * Copyright 2017 Red Hat, Inc.
+ * All Rights Reserved.
+ *
+ * Export of this software from the United States of America may
```

Comment:
If the intent is to use the MIT krb5 license here, please use the 2-clause BSD one from src/prototype/prototype.c.